appconfig/settings/rdb.q

```diff
@@ -10,13 +10,11 @@ tickerplanttypes:`segmentedtickerplant
 gatewatypes:`none
 replaylog:1b
 
-hdbtypes:()			//connection to HDB not needed
+hdbtypes:()			            //connection to HDB not needed
 
 subfiltered:0b
 // path to rdbsub{i}.csv
-subcsv:hsym first `.proc.getconfigfile["rdbsub/rdbsub",(procnum:3_string .proc`procname),".csv"]
-// save the subcsv file onto disk
-subcsv 0: csv 0: flip`tabname`filters`columns!(`trade`quote;2#`$"sym in .ds.stripe[sym;",(string -1+value procnum),"]";`)
+subcsv:hsym first `.proc.getconfigfile["rdbsub/rdbsub.csv"]
```

Comment:
Could we just make this

```suggestion
subcsv:hsym first `.proc.getconfigfile["rdbsub/",string[.proc.procname],".csv"]
```

and then all we have to do for each process is create a new .csv? Rather than having to have a separate `.q` script for each rdb as well.

Would need to rename the default one to `rdb1.csv` but I think that's fine. Obviously would also have to update the docs again, but I think it just simplifies things a little?

Reply:
```suggestion
subcsv:hsym first `.proc.getconfigfile["rdbsub/rdbsub",(3_string .proc`procname),".csv"]
```
How about this previous one? Just without the saving to disk. So this creates rdbsub1.csv. Will update the docs if this is ok.

Reply:
it seems simpler/cleaner to me to just use the procname without manipulating it, but I think either is fine :+1: